.test-infra/jenkins/job_Publish_Docker_Snapshots.groovy

```diff
@@ -42,10 +42,9 @@ job('beam_Publish_Docker_Snapshots') {
       rootBuildScriptDir(commonJobProperties.checkoutDir)
       commonJobProperties.setGradleSwitches(delegate)
       SUPPORTED_CONTAINER_TASKS.each { taskVer ->
-        tasks(":sdks:python:container:${taskVer}:dockerPush")
+        tasks(":sdks:python:container:${taskVer}:dockerTagPush")
```

Comment:
I still don't quite get the difference between these two, but why do we need to update the python task? Isn't it just failing on the Go SDK/can we just remove the go one?

Same question about flink as well.

Reply:
Whoops Good catch. This was seeing if it would fix the Go build somehow. It did not and I didn't revert it. Thanks!